demos/python_demos/face_recognition_demo/face_detector.py

```diff
@@ -46,14 +46,19 @@ def clip(self, width, height):
             self.position[:] = clip(self.position, min, max)
             self.size[:] = clip(self.size, min, max)
 
-    def __init__(self, model, confidence_threshold=0.5, roi_scale_factor=1.15):
+    def __init__(self, model, confidence_threshold=0.5, roi_scale_factor=1.15,input_h=0, input_w=0):
         super(FaceDetector, self).__init__(model)
 
         assert len(model.inputs) == 1, "Expected 1 input blob"
         assert len(model.outputs) == 1, "Expected 1 output blob"
         self.input_blob = next(iter(model.inputs))
         self.output_blob = next(iter(model.outputs))
-        self.input_shape = model.inputs[self.input_blob].shape
+        #add reshape function to detect more small faces in image
+        if (input_h and input_w ):
+            self.input_shape = [1, 3, input_h, input_w]
+        else:
+            self.input_shape = model.inputs[self.input_blob].shape
+        print("face detector input shape:", self.input_shape)
```

Comment:
* As for me, this changes are not required, because `model.inputs[self.input_blob].shape` changes when reshape is called.

* The demo uses `log` instead of `print`. Is it useful to print the values that a user specified? They just passed to print unchanged (well, concatenated with [1, 3]).

Reply:
yes, this change not need after reshape is called.  I will revert this change. thanks